README.md

```diff
@@ -1,91 +1,189 @@
 # CSS Media Queries
 
-## Overview
-In this lesson we will look at using CSS3 Media Queries to detect viewport sizes and adjust CSS styles accordingly.
+## Problem Statement
+
+A component of designing and building responsive layouts is being mindful
+of the wide variety of screen sizes. Originally, media queries could be used
+to set specific breakpoints based on a specific set of devices. As the market
+of web-connected electronics has grown, media queries have evolved to be more 
+inclusive of new sizes and aspect ratios. What are media queries, exactly, and 
+how can they be taken advantage of in building page layouts?
```

Comment:
In this para. I don't understand the "Originally" to "current" difference. Originally implies a stark difference: e.g.

> Originally I ate worms, but now I prefer a pizza

Is "Originally" really needed to make this paragraph work? Or is there a vital contrast that I'm not picking up?